### PR TITLE
Simplifiy document format.

### PR DESCRIPTION
**Pull request by Alexander-Miller**

### CODE REVIEW

File: livingdoc-repositories/src/main/kotlin/org/livingdoc/repositories/Document.kt [hunk 1]
```diff
@@ -1,37 +1,16 @@
...
+
+abstract class ResultItem(val result: Result = Result.Unknown)
+
+data class DecisionTable(val headers: List<String>, val rows: List<DecisionTableRow>) : ResultItem()
+
+data class DecisionTableRow(val cells: Map<String, DecisionTableCell>)
+
+data class DecisionTableCell(val text: String) : ResultItem()
+
+data class DecisionList(val items: List<String>) : ResultItem()
```
Comment (slu-it) [inline livingdoc-repositories/src/main/kotlin/org/livingdoc/repositories/Document.kt]: There is no such thing as a `DecisionList` bullet points / numbered lists are scenarios.

See als the [Glossary](https://github.com/LivingDoc/livingdoc/blob/master/doc/glossary.adoc)...

Comment (Alexander-Miller) [reply livingdoc-repositories/src/main/kotlin/org/livingdoc/repositories/Document.kt]: done

File: livingdoc-repositories/src/main/kotlin/org/livingdoc/repositories/Document.kt [hunk 2]
```diff
@@ -1,37 +1,16 @@
...
+
+abstract class ResultItem(val result: Result = Result.Unknown)
+
+data class DecisionTable(val headers: List<String>, val rows: List<DecisionTableRow>) : ResultItem()
+
+data class DecisionTableRow(val cells: Map<String, DecisionTableCell>)
+
+data class DecisionTableCell(val text: String) : ResultItem()
+
+data class Scenario(val items: List<String>) : ResultItem()
```
Comment (slu-it) [inline livingdoc-repositories/src/main/kotlin/org/livingdoc/repositories/Document.kt]: items >> steps

Comment (Alexander-Miller) [reply livingdoc-repositories/src/main/kotlin/org/livingdoc/repositories/Document.kt]: done

**Pull request merged**